tbx/project_styleguide/templates/patterns/base_page.html

```diff
@@ -57,12 +57,11 @@
         {% endif %}
     {% else %}
         {# Use default social sharing image #}
-        {# TODO Update social images #}
-        {# {% with page.theme_class|default:'theme-coral' as image_theme %} #}
-        {#     {% with 'images/social/twitter-'|add:image_theme|add:'.webp' as twitter_default_image %} #}
-        {#         <meta name="twitter:image" content="{{ current_site.root_url }}{% static twitter_default_image %}"> #}
-        {#     {% endwith %} #}
-        {# {% endwith %} #}
+        {% with page.theme_class|default:'theme-coral' as image_theme %}
+            {% with 'images/social/twitter-'|add:image_theme|add:'.webp' as twitter_default_image %}
```

Comment:
Suggestion: Once this goes onto staging it's worth checking that this woks on Whatsapp/Slack as I remember Nick L having an issue with it, possibly due to it being webp but that was a while ago

Reply:
There's a slack thread on this - let me know if you want me to dig it out!